### DISCUSSION TITLE
Is it possible to move server.ts into subfolder 'src'

### DISCUSSION THREAD

**Discussion by jgoblwr** [adonisjs/core#2363] [Help] [answered]
Hi,

I am working with Adonis 5 preview version on windows 10 machine.

I would like to have most of the code related files for a project in a src subfolder...ie. src/app, src/commands, src/providers , src/server.ts. From what I could tell maybe this is possible by setting some config in the .adonisrc.json file. But there does not seem to be a config setting in it for where the server.ts file will be. And when I try run "node ace serve --watch" I get the following error...

Error: Cannot find module 'C:\dev\onboarding-service\server.ts'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
[ warn ]  Underlying HTTP server died with "1 code"

How can I configure to have code files including server.ts file in a subfolder?

Many thanks

John Gobl

**Comment by RomainLanz** [comment, marked as answer]
Hey @jgoblwr! 👋 

There is no current way of changing that.

**Reply by jgoblwr** [reply]
Hi @RomainLanz . 

Ok, thanks for taking the time to respond and clarifying this.

Many thanks,

John Gobl
